integration-tests/tests/index.test.ts

```diff
@@ -10,7 +10,7 @@ import {
   newClient,
 } from '@cipherstash/protect-ffi'
 
-const encryptConfig = JSON.stringify({
+const encryptConfig = {
```

Comment:
Encrypt config is now passed in as an object instead of a string.

Reply:
Hell yeah that's huge